docs/contributing/how-to-write-a-reference-guide.md

```diff
@@ -143,7 +143,7 @@ Guide overview articles are essentially new parent categories that help organize
 1. [Stub articles](/contributing/stub-list/) (docs that already exist on the site but don't have content in them yet)
 2. Article sections with the [help wanted and type:documentation](https://github.com/gatsbyjs/gatsby/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22+label%3A%22type%3A+documentation%22) labels on GitHub
 3. Article sections related to improving [key learning workflows](https://github.com/gatsbyjs/gatsby/issues/13708), like "e-commerce"
-4. Article sections listed in the "Backlog" or "To prioritize" sections of the [Learning / Devrel Roadmap](https://github.com/gatsbyjs/gatsby/projects/10) on GitHub
+4. Article sections listed in the "Backlog" or "To prioritize" sections of the [Documentation / Devrel Roadmap](https://github.com/gatsbyjs/gatsby/projects/10) on GitHub
```

Comment:
We mostly stopped using this project when we moved to Clubhouse. We can probably remove this item entirely.